test/check-versions-whitespace.ts

```diff
@@ -0,0 +1,23 @@
+import assert from 'assert';
+import fs from 'fs';
+import path from 'path';
+
+const versionsFilePath = path.join(__dirname, '../configs/versions.json');
+
+describe('check versions.json whitespace', () => {
+  it('whitespace exists', () => {
+    const versionsJsonString = fs.readFileSync(versionsFilePath, 'utf-8');
```

Comment:
I'm guessing importing json strips the whitespace so that's why we need to use fs.read?